packages/react/src/queries/fees.ts

```diff
@@ -0,0 +1,20 @@
+export function getProtocolAndPlatformsFees(
```

Comment:
all queries need to be inside the client

Reply:
I have opened PR #12 
It will make it so that all queries are within the client 😄